contracts/tasks/helpers/Prover.ts

```diff
@@ -156,13 +165,12 @@ export class Prover {
 
       this.validateCommitment(circuitInputs.currentSbCommitment as BigNumberish, currentSbCommitmentOnChain);
 
-      const coordPubKeyHashOnChain = BigInt(coordinatorPubKeyHash);
       const coordPubKeyHashOffChain = hashLeftRight(
-        BigInt((circuitInputs.coordPubKey as BigNumberish[])[0]),
-        BigInt((circuitInputs.coordPubKey as BigNumberish[])[1]),
+        BigInt((coordinatorPubKey as BigNumberish[])[0]),
```

Comment:
this check is also redundant, will need to check circuitInputs.coordinatorPubKeyHash with the on chain one